azure-pipelines.yml

```diff
@@ -4,21 +4,16 @@ trigger:
     include:
       - "*"
 
-pr: none
-
-workspace:
-  clean: all
```

Comment:
We've removed workspace clean here but not added it in either the stage or job.  Do we not want to have workspace clean in by default?

Reply:
Added